plugin/converter/jsonnet/jsonnet.go

```diff
@@ -0,0 +1,57 @@
+package jsonnet
+
+import (
+	"bytes"
+
+	"github.com/drone/drone/core"
+
+	"github.com/google/go-jsonnet"
+)
+
+func Parse(req *core.ConvertArgs, template *core.Template, templateData map[string]interface{}) (string, error) {
+	// create the jsonnet vm
+	vm := jsonnet.MakeVM()
+	vm.MaxStack = 500
+	vm.StringOutput = false
+	vm.ErrorFormatter.SetMaxStackTraceSize(20)
+
+	var jsonnetFile string
+	var jsonentFileName string
+	if template != nil {
+		jsonnetFile = template.Data
+		jsonentFileName = template.Name
+	} else {
+		jsonnetFile = req.Config.Data
+		jsonentFileName = req.Repo.Config
+	}
+	// map external inputs
+	if len(templateData) != 0 {
+		for k, v := range templateData {
+			if s, ok := v.(string); ok {
+				key := "input." + k
```

Comment:
for now, you could probably do this:

```
for k, v := range templateData {
  key := fmt.Sprintf("input." + k)
  val := fmt.Sprint(v)
  vm.ExtVar(key, val)
}
```

the `fmt.Sprint` command will handle conversion of all types to a string

Reply:
Good to know, updated.